package.py

```diff
@@ -705,10 +732,10 @@ def commands_step(path, commands):
                         pwd=os.getcwd()
                     ))
                 runtime = query.runtime
-                if runtime.startswith('python'):
-                    pip_requirements_step(
-                        os.path.join(path, 'requirements.txt'))
-                step('zip', path, None)
+                if runtime.startswith("python"):
+                    pip_requirements_step(os.path.join(path, "requirements.txt"))
```

Comment:
`pip_requirements_step()` already appends the filename if `path` is a directory: 
```suggestion
                    pip_requirements_step(path)
```